features/formatter/pretty.feature

```diff
@@ -23,6 +23,181 @@ Feature: pretty formatter
       0s
     """
 
+  Scenario: Support of Feature Plus Background and Scenario Node
+    Given a feature "features/simple.feature" file:
+    """
+        Feature: simple feature
+            simple feature description
+        Background: simple background
+            Given passing step
+            And passing step
+        Scenario: simple scenario
+            simple scenario description
+    """
+    When I run feature suite with formatter "pretty"
+    Then the rendered output will be as follows:
+    """
+      Feature: simple feature
+        simple feature description
+
+        Background: simple background
+            Given passing step
```

Comment:
I'm confused by this indentation - do we indent the steps by 4 characters within a Background?

Reply:
Or is this about having a universal fixed indentation for every step irrespective of whether it's inside a Rule, Background, Scenario etc?

Reply:
I think the indentation is actually ignored in the tests. I changed indentation and it did not caused the tests to fail.